src/components/forms/formComponents.js

```diff
@@ -485,6 +485,7 @@ export const FormInputCheckbox = (config) => {
     label({
       className: `regular-checkbox ${!isValid(validation) ? 'errored' : ''}`,
       htmlFor: `${id}`,
+      style: disabled ? { cursor: 'not-allowed' } : null
     }, [toggleText])
   ]);
```

Comment:
I'm seeing two things unrelated to the ticket that shouldn't hold up this PR, maybe we can address separately.
1. I had the question of _Why is the input not inside the label?_ and needed to look that up. I learned HTML in the 90's, but apparently it can be [before, around, or after](https://www.w3.org/TR/html401/interact/forms.html#edef-LABEL)
2. We're still using hyperscript here ... would be really nice to get rid of that and stick with jsx.

Reply:
1. Yes! And I also don't understand why the input tag is not directly creating the checkbox.
2. Definitely agree!